src/devices/Bmp180/Bmp180.csproj

```diff
@@ -0,0 +1,19 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Library</OutputType>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <ApplicationIcon />
+    <StartupObject />
+    <LangVersion>7.3</LangVersion>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
+    <PlatformTarget>AnyCPU</PlatformTarget>
+  </PropertyGroup>
+
+  <ItemGroup>

```

Comment:
The way you have this project setup, it will include the sources from the samples project as well for your main library which is not what you want. I would suggest you taking a look at an existing binding and try to mimic what we do in those projects. Specifically, you should set the property `EnableDefaultItems` to false and then manually add the `Compile` item for each of your .cs files like:

[https://github.com/dotnet/iot/blob/0a4a036ef8404aa68b9437655690679bd53358c0/src/devices/Mcp3008/Mcp3008.csproj#L1-L14](https://github.com/dotnet/iot/blob/0a4a036ef8404aa68b9437655690679bd53358c0/src/devices/Mcp3008/Mcp3008.csproj#L1-L14)